src/mapping/mod.rs

```diff
@@ -306,410 +306,3 @@ impl Function for LogFn {
         Ok(())
     }
 }
-
-//------------------------------------------------------------------------------
```

Comment:
Did you mean to drop this code?

Reply:
Yeah, I intended for the scope of this PR to be bigger, but backed out of that. This can still be removed, as it's dead code (the `src/mapping` directory still contains the _old_ remap implementation).